net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="A small dynamic dns client with multiple servers support and a high flexibility"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
+CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+SOURCE_DIR="yaddns-yaddns-$portVersion"
+PATCHES="yaddns-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	yaddns = $portVersion
+	cmd:yaddns = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autom4te
+	cmd:automake
+	cmd:autoreconf
+	cmd:g++
+	cmd:gcc
+	cmd:git
+	cmd:ld
+	cmd:make
+	cmd:makeinfo
+	"
+
+USER_SETTINGS_FILES="
+	settings/yaddns/yaddns.conf template data/yaddns/yaddns.conf.example
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/yaddns/yaddns.conf keep-old
+	"
+
+BUILD()
+{
+	NOCONFIGURE=1 ./autogen.sh
```

Comment:
```diff
-	NOCONFIGURE=1 ./autogen.sh
+	./autogen.sh
```
(`NOCONFIGURE=1` does not change anything on yaddns's `autogen.sh` so we can skip it.)
Well, I'm almost sure about that.

@alaviss, what would you say? Here's what we have in the `autogen.sh`:
```
#! /bin/sh
aclocal \
&& automake --add-missing \
&& autoconf
```